### DISCUSSION TITLE
Change the installer logic

### DISCUSSION THREAD

**Discussion by Sichroteph** [OnionUI/Onion#638] [Ideas]
Just an idea as going into apps is quite recommended to enjoy all the onion features.
We could honestly stay like this.

1)  The Start button in the installer could instead of launch right away the procedure :
Make the installer switch to apps -> expert -> installation

or 2) The installer display a little panel if the user choose to install right away without going into apps 

or 3) The "start" icon is displayed only once the user went at least once in the apps panel

**Comment by Aemiii91** [comment]
I like it 👍
